Use window.ethereum to get vETH2 total supply

exitonoff opened the pull request:
Landing page used Etherscan to fetch vETH2 total supply. That call was failing so changed the fetch to use window.ethereum. If it doesn't exist, fallback values are used.
Those fallback values should maybe be updated to be a bit closer to the current values?

sharedStake-main commented:
As a side note, I suggest to use a second fall-back with the etherscan link.

exitonoff commented:
> As a side note, I suggest to use a second fall-back with the etherscan link.

So leave Etherscan API call as a fallback, even though it exposes your API key? Just wanting to make sure I got what you are saying.

sharedStake-main commented:
@exitonoff yes, codeflow should be:
- Etherscan API call, it is okay if there is a DDOS attack, we can change it.
- window.ethereum fallback, in c oase of a fail of ES API call
- fallback values

Does it sound good?

exitonoff commented:
@sharedStake-main 

I changed the code to primarily use Etherscan API. Secondary we use window.ethereum and if it doesn't exist we fallback to hard coded values.

I went ahead and separated the code into own functions to make the code bit easier to read and to make the code bit more fault resilient. Also updated the hard coded TVL to represent the locked ETH amount bit more accurately :D
